achievements/666.achievement.js

```diff
@@ -0,0 +1,38 @@
+var sixes = {
+  name: 'I am the devil I like metal',
+  check: function(pullRequest, shall) {
+    if (isPrime(pullRequest.number)) {
```

Comment:
change func name to hasSix

Reply:
See my comment on the function name below 👇